pkg/adapter/reconciler/blueprint_controller.go

```diff
@@ -25,18 +25,16 @@ func NewBlueprintReconciler(clientSet ecosystemClientSet, recorder eventRecorder
 
 // Reconcile is part of the main kubernetes reconciliation loop which aims to
 // move the current state of the cluster closer to the desired state.
-// TODO(user): Modify the Reconcile function to compare the state specified by
-// the Blueprint object against the actual cluster state, and then
-// perform operations to make the cluster state reflect the state specified by
-// the user.
 //
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.16.3/pkg/reconcile
 func (r *BlueprintReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
-
-	// TODO(user): your logic here
-
+	logger := log.FromContext(ctx)
+	err := r.blueprintChangeHandler.HandleBlueprintSpecChange(ctx, req.Name)
+	if err != nil {
+		logger.Error(err, "error in blueprint reconciliation loop with blueprint", "resourceName", req.Name)
```

Comment:
```suggestion
		logger.Error(err, "error in blueprint reconciliation loop", "resourceName", req.Name)
```
This is kinda redundant.